index.js

```diff
@@ -1 +1,82 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db')
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/users',(req, res) => {
+    const {name , bio} = req.body;
+
+    db.insert(name && bio)
```

Comment:
I would make this an object that has name and bio passed in...i.e. `db.insert({name, bio})`